gradle.properties

```diff
@@ -56,10 +57,10 @@ marathonClientVersion=0.6.2
 micrometerVersion=1.2.0
 mockitoVersion=3.3.3
 jacocoVersion=0.8.5
-nettyVersion=4.1.50.Final
-nettyBoringSSLVersion=2.0.30.Final
-protobufGradlePlugin=0.8.9
-protobufProtocVersion=3.13.0
+nettyVersion=4.1.59.Final
+nettyBoringSSLVersion=2.0.36.Final
```

Comment:
Let's coordinate with #5747 which is also upgrading these libraries. That PR upgrades them to smaller versions though.

Reply:
@andreipaduroiu Did you mean #6090?

Reply:
Yes. Sorry about that.